app/views/waste_carriers_engine/shared/_address_display.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= a.firstName %>  <%= a.lastName %></br>
+<% unless a.houseNumber.blank? %>
```

Comment:
A couple thoughts on this:

- These attributes are aliased to use `snake_case`, so would prefer `house_number` to `houseNumber`
- I'd prefer `if foo.present?` instead of `unless foo.blank?`, but I think both might be unnecessary because:
- I recently did some refactoring to reduce duplication in displaying addresses and now we have this `displayable_address` helper method: https://github.com/DEFRA/waste-carriers-renewals/blob/master/app/helpers/waste_carriers_engine/application_helper.rb#L55-L67 Could we use this instead and just display each line of the array returned? I think it might simplify things a bit.